examples/cg/cg.py

```diff
@@ -0,0 +1,90 @@
+import argparse
+import contextlib
+import time
+
+import numpy as np
+import six
+
+import cupy
+
+
+@contextlib.contextmanager
+def timer(message):
+    cupy.cuda.Stream.null.synchronize()
+    start = time.time()
+    yield
+    cupy.cuda.Stream.null.synchronize()
+    end = time.time()
+    print('%s:  %f sec' % (message, end - start))
+
+
+def fit(A, b, x0, tol, max_iter):
```

Comment:
I don't think we need to feed `x0` as an argument because it is always a zero-array.